formspree/templates/forms/list.html

```diff
@@ -35,6 +35,9 @@
           <a href="{{ url_for('form-submissions', hashid=form.hashid) }}">{{ form.counter }} submissions</a>
         {% endif %}
       </td>
+		<td>
+		<a href="{{ url_for('form-deletion', hashid=form.hashid) }}">Delete</a>
+		</td>
```

Comment:
Yep I wasn't sure about this. I added these methods and said they needed to be logged in to perform them and then if they are logged in they need to actually be the form owner in order to delete. But I wasn't really sure between a GET and POST request since I guess you could say we're posting data but I was POSTing through the URL unless you think I should send that data through a form and have the hashid and submission id be the data that was posted.

Thank you very much for your other points also! I wasn't really sure about that stuff, so good to know the correct syntax.